src/ServiceControl.Transports.ASQ/ServiceControl.Transports.ASQ.csproj

```diff
@@ -10,10 +10,10 @@
 
   <ItemGroup>
     <PackageReference Include="NServiceBus.Transport.AzureStorageQueues" Version="10.0.4" />
-    <PackageReference Include="System.Text.Json" Version="6.0.2" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
+    <PackageReference Include="System.Text.Json" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" TreatAsUsed="true" />
```

Comment:
Wasn't sure about TreatAsUsed="true" is required.

Reply:
it prevents analysis tools to warn about them being unused so I think we should use them